components/ui/icons/src/main/res/drawable/mozac_ic_addons_extensions.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!-- This Source Code Form is subject to the terms of the Mozilla Public
+   - License, v. 2.0. If a copy of the MPL was not distributed with this
+   - file, You can obtain one at http://mozilla.org/MPL/2.0/. -->
+<vector xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportWidth="24"
+    android:viewportHeight="24">
+  <path
+      android:pathData="m20.125,12c-1.213,0 -1.25,1.25 -2.187,1.25 -0.5138,-0.0101 -0.9279,-0.4242 -0.938,-0.938v-4.062c0,-0.6904 -0.5596,-1.25 -1.25,-1.25h-4.062c-0.5138,-0.0101 -0.9279,-0.4242 -0.938,-0.938 0,-0.937 1.25,-0.974 1.25,-2.187 0,-1.081 -1.119,-1.875 -2.5,-1.875s-2.5,0.794 -2.5,1.875c0,1.213 1.25,1.25 1.25,2.187 -0.0101,0.5138 -0.4242,0.9279 -0.938,0.938h-4.062c-0.3315,0 -0.6495,0.1317 -0.8839,0.3661 -0.2344,0.2344 -0.3661,0.5524 -0.3661,0.8839v2.812c0.0101,0.5138 0.4242,0.9279 0.938,0.938 0.937,0 0.974,-1.25 2.187,-1.25 1.081,0 1.875,1.119 1.875,2.5s-0.794,2.5 -1.875,2.5c-1.213,0 -1.25,-1.25 -2.187,-1.25 -0.5138,0.0101 -0.9279,0.4242 -0.938,0.938v5.312c0,0.6904 0.5596,1.25 1.25,1.25h4.062c0.5138,-0.0101 0.9279,-0.4242 0.938,-0.938 0,-0.937 -1.25,-0.974 -1.25,-2.187 0,-1.081 1.119,-1.875 2.5,-1.875s2.5,0.794 2.5,1.875c0,1.213 -1.25,1.25 -1.25,2.187 0.0101,0.5138 0.4242,0.9279 0.938,0.938h4.062c0.6904,0 1.25,-0.5596 1.25,-1.25v-4.062c0.0101,-0.5138 0.4242,-0.9279 0.938,-0.938 0.937,0 0.974,1.25 2.187,1.25 1.081,0 1.875,-1.119 1.875,-2.5s-0.794,-2.5 -1.875,-2.5z"
+      android:fillColor="@color/mozac_ui_icons_fill"
```

Comment:
Should we use `?primaryText` instead? I think this what most other icons in the toolbar menu are using for supporting dark/light themes :)

Reply:
Sorry, my mistake.